.github/workflows/unit-test.yml

```diff
@@ -375,7 +375,8 @@ jobs:
         runs-on: ubuntu-latest
         permissions:
             contents: read
-        if: ${{ github.repository == 'WordPress/gutenberg' || github.event_name == 'pull_request' }}
+        if: false
+        #if: ${{ github.repository == 'WordPress/gutenberg' || github.event_name == 'pull_request' }}
```

Comment:
Should we add a comment explaining why this is disabled, probably linking to https://github.com/WordPress/gutenberg/issues/71336?

Reply:
Good call - added in https://github.com/WordPress/gutenberg/pull/71337/commits/cb86ff8e5d96f8bb71e7cb82b325c3d8709c4f56